File: benchmark/results/java-invocation-inline/run-005/b-baseline/output/com/metaengine/demo/notification/value_objects/Recipient.java
package com.metaengine.demo.notification.value_objects;

import java.time.Instant;

/** Recipient value object — addressed party for a notification. */
public record Recipient(
    String id,
    Instant createdAt,
    Instant updatedAt,
    String name,
    String description
) {}

File: benchmark/results/python-multilang/run-002/b-baseline/output/metaengine_demo/notification/value_objects/recipient.py
"""Recipient value object for the notification domain."""

from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class Recipient:
    """Immutable description of a notification recipient."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str
